anvil-rstudio-base/README.md

```diff
@@ -1,19 +1,76 @@
+| 0.0.1 | 10/06/2019 | Adrian Sharma: adrian@broadinstitute.org |
 
-## Base image
-* [rocker/tidyverse](https://hub.docker.com/r/rocker/tidyverse/)
+# AnVIL RStudio Docker Image
 
-## Applications
-* [RStudio Server](https://www.rstudio.com/products/rstudio-server/)
+[Rstudio](https://rstudio.com/products/rstudio/) is an integrated development environment to support the R statistical programming language. The AnVIL RStudio Docker is a version-tagged image of RStudio, providing consistent RStudio software, packages, and dependencies with every use. This document describes the [RStudio](https://rstudio.com/products/rstudio/) Docker Image and how to use it in the cloud-based platform [Terra](app.terra.bio). Unlike the currently existing [Terra base images](https://github.com/DataBiosphere/terra-docker#terra-base-images), RStudio is not hosted in a Jupyter Notebook; any work performed in Terra RStudio will not be saved. At the end of this document, you will find Instructions and suggestions for saving RStudio data and code. 
+
+## Table of Contents
+- [RStudio Docker Contents](#rstudio-docker-contents)
+  * [Accessing the AnVIL RStudio Image](#accessing-the-anvil-rstudio-image)
+  * [Base Image](#base-image)
+  * [Applications](#applications)
+  * [Libraries](#libraries)
+  * [Customizations](#customizations)
+- [Using R Studio in Terra](#using-r-studio-in-terra)
+- [Saving RStudio Data](#saving-rstudio-data)
+- [Versioning](#versioning)
+
+## RStudio Docker Contents
+
+### Accessing the AnVIL RStudio Image
+
+The AnVIL Rstudio Image can be accessed using the following URL: us.gcr.io/anvil-gcr-public/anvil-rstudio-base:0.0.1. 
+
+### Base Image
+
+The base image for the AnVIL RStudio Docker is the [rocker/tidyverse](https://hub.docker.com/r/rocker/tidyverse/) image provided by the [Rocker Project](https://www.rocker-project.org/). This image additionally contains tidyverse and devtools.
+
+
+### Applications
+
+The AnVIL R Docker contains the [RStudio Server](https://www.rstudio.com/products/rstudio-server/) application which supports a web browser version of RStudio. 
+
+### Libraries
+
+The following packages are pre-loaded in the AnVIL RStudio image:
+
+* [tidyverse](https://www.tidyverse.org/packages/): a suite of packages designed for data sciences 
+* [google-cloud-sdk](https://cloud.google.com/sdk/): tools for the google cloud platform
+* [googleCloudStorageR](http://code.markedmondson.me/googleCloudStorageR/): an R library for interacting with google cloud storage
+* [bigrquery](https://github.com/r-dbi/bigrquery): a package to allow interaction with Google's BigQuery
+* [DataBiosphere/Ronaldo](https://github.com/DataBiosphere/Ronaldo): a package of functions commonly used for R Notebooks created from Leonardo
+
+### Customizations
+
+The AnVIL RStudio image is customized to disable the RStudio login screen. Future customizations will include:
+
+* RStudio hooks to refresh credentials 
+* An RStudio pause after inactivity
```

Comment:
We currently do this for jupyter notebooks, and we call it `auto pause`, I think we should use the same terminology

Reply:
Done